boards/arm/lpcxpresso55s69/doc/index.rst

```diff
@@ -78,10 +78,11 @@ Other hardware features are not currently enabled such as dual core or secure/no
 Targets available for this board are:
 
 - *lpcxpresso55s69_cpu0* secure (S) address space for CPU0
-- *lpcxpresso55s69_ns* non-secure (NS) address space for CPU0
+- *lpcxpresso55s69_nonsecure* non-secure (NS) address space for CPU0
```

Comment:
Is _nonsecure really the convention @ioannisg ? I don't think I've seen this anywhere else.

Reply:
@MaureenHelm That's what we used with the an521 (`mps2_an521_nonsecure`), so I proposed keeping that to @agansari, but if you'd prefer appending something shorter we can change the AN521 build target as well.

Reply:
I like the idea of getting them aligned. I think all the nordic boards use ns.

Reply:
Yeap, that's right, the ns (without underscore) what Nordic and MuSca boards use.